src/main/java/org/spongepowered/api/scheduler/Task.java

```diff
@@ -219,6 +219,13 @@ default Builder execute(Runnable runnable) {
          */
         Builder name(String name);
 
+        /**
+         * Builds the task.
+         *
+         * @return The task
+         */
+        Task build();
```

Comment:
What? What about submitting it?

Reply:
See the deprecation right below this method.

Reply:
Why is that necessary? If I already started making a sync'ed task from the server scheduler, shouldn't it be possible to just call submit? Why clog up code to force me to submit it after the fact?

Or, maybe make t build method as such: `submit(Object plugin, Scheduler scheduler)`.

Reply:
@gabizou 

No the submit stuff made the Scheduler API foreign to other builder-like objects and no one ever went back to fix it.

There is no reason why you shouldn't be able to create a Task that is not submitted yet and no I WANT you to express intent on the scheduler you want to submit your task too.

Reply:
I can agree to that. On the other hand, the main reason why we moved away from registering catalog types in build methods like this was because we have a registry event for registration of those catalog types. As tasks are not registered in the same way, I had the feeling that this would make sense to have a differing way for building and submitting them. 

Would you then say that an asynchronous task would be regardless of the platform for submitting the task? Or would you still define the platform for submitting the task?